src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -315,6 +315,10 @@ function ReimbursementAccountPage({reimbursementAccount, route, onfidoToken, pol
 
             const currentStepRouteParam = getStepToOpenFromRouteParams(route);
             if (currentStepRouteParam === currentStep) {
+                // If the user is connecting online with plaid, reset any bank account errors so we don't persist old data from a potential previous connection
+                if (currentStep === CONST.BANK_ACCOUNT.STEP.BANK_ACCOUNT && achData.subStep === CONST.BANK_ACCOUNT.SETUP_TYPE.PLAID && !shouldShowContinueSetupButton) {
+                    BankAccounts.hideBankAccountErrors();
+                }
```

Comment:
```suggestion
                }

```
NAB - don't we like having blank lines above comments, or is that just PHP? (Same with L318 too)

Reply:
I think we want a new line here but not on L318 because it's the beginning of a block.